app/assets/javascripts/cable.js

```diff
@@ -0,0 +1,13 @@
+// Action Cable provides the framework to deal with WebSockets in Rails.
```

Comment:
There is, because they're added by default by the Rails new app generator. It will make comparing / diffing for the next upgrades easier.